packages/core/admin/admin/src/pages/SettingsPage/pages/ApiTokens/EditView/utils/getDateOfExpiration.js

```diff
@@ -0,0 +1,6 @@
+import { addDays, format } from 'date-fns';
+
+const getDateOfExpiration = (createdAt, duration) =>
+  format(addDays(new Date(createdAt), duration), 'PPP');
```

Comment:
I think it would be nice to add some tests for this utils 😊

Reply:
agreed will do so